buildpacks/sbt/tests/integration_tests.rs

```diff
@@ -1,117 +0,0 @@
-use libcnb_test::{
```

Comment:
I think the integration tests will now be run in series (each file in series that is), whereas before all tests would have been run in parallel. Have you confirmed the impact on end to end time? (The integration tests on `main` for this repo already take 24-27 mins as it is.)

To have the tests be in separate files, but have each of the files be run separately requires nesting one level deeper, like the example here:
https://matklad.github.io/2021/02/27/delete-cargo-integration-tests.html#Implications

...or else using the trick that Python uses, which is to import them as unit tests instead:
https://github.com/heroku/buildpacks-python/blob/7a88302e7f53b1bd9935e164958b94f54365009a/src/main.rs#L156-L162

Reply:
I plan to revise the integration test strategy for the repo as a quick follow-up. Basically this to get them in, then another PR that will improve CI time for all integration tests in this repo (where similar issues exist already). I almost put those in this PR, but decided to split it up).